### PR TITLE
kirin: connector IRE removed

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVP-1341

### CODE REVIEW

File: tests/integration/test_end_point.py [hunk 1]
```diff
@@ -37,12 +37,6 @@
 import pytest
 
 
-def test_end_point():
```
Comment (mbounabi) [inline tests/integration/test_end_point.py]: Shouldn't there be a test with 'status' and 'cots' in the response?

Comment (kadhikari) [reply tests/integration/test_end_point.py]: it's my mistake, i will add it

File: kirin/default_settings.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 NAVITIA_TOKEN = os.getenv('KIRIN_NAVITIA_TOKEN', None)
 
-CONTRIBUTOR = os.getenv('KIRIN_CONTRIBUTOR', 'realtime.ire')
+CONTRIBUTOR = os.getenv('KIRIN_CONTRIBUTOR', 'realtime.cots')
```
Comment (pbougue) [inline kirin/default_settings.py]: COTS_CONTRIBUTOR already manages cots ;)
We must remove that var (from everywhere, including fab & conf).

Please, can you also check (if you didn't already) that there are no other IRE-specific params ?
(for example, I think `KIRIN_NAVITIA_INSTANCE` is still used by COTS, to be confirmed. If not it should disappear.)

File: kirin/resources.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ class Index(Resource):
     def get(self):
         response = {
             'status': {'href': url_for('status', _external=True)},
-            'ire': {'href': url_for('ire', _external=True)},
+            'gtfs_rt': {'href': url_for('gtfs_rt', _external=True)},
```
Comment (pbougue) [inline kirin/resources.py]: I don't think we want to expose that, it's for debug purpose only as gtfs-rt is polled, not posted.

File: tests/gtfs_rt_test.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def unix_time(y, m, d, h, min, s):
     return calendar.timegm(d.utctimetuple())
 
 def make_96231_20150728_0():
-    """gtfs rt feed corresponding to train_96231_delayed.xml"""
+    """gtfs rt feed corresponding to cots_train_96231_delayed.json"""
```
Comment (mbounabi) [inline tests/gtfs_rt_test.py]: No changes needed in the rest of the test? 'train_96231_delayed.xml for IRE was exactly the same as 'cots_train_96231_delayed.json' for COTS?

File: tests/integration/handle_test.py [hunk 5]
```diff
@@ -77,7 +77,7 @@ def setup_database():
                                      {'id': 'sa:3', 'departure': None, 'arrival': _dt("10:05"),
                                       'departure_status': 'none', 'arrival_status': 'none'},
                                  ])
-        rtu = RealTimeUpdate(None, 'ire', contributor='realtime.ire')
+        rtu = RealTimeUpdate(None, 'cots', contributor='realtime.cots')
```
Comment (mbounabi) [inline tests/integration/handle_test.py]: General comment about the file:
Does that mean that previously, there weren't any tests COTS for every test cases in this file ? Or does the creation of the RealTimeUpdate doesn't affect the test?

File: tests/gtfs_rt_test.py (hunk 4 above)
Comment (pbougue) [reply tests/gtfs_rt_test.py]: As far as I remember, we designed cots-tests (fixtures) to use the same python test-methods than ire-tests.
So it should be alright yes.

File: tests/integration/handle_test.py (hunk 5 above)
Comment (pbougue) [reply tests/integration/handle_test.py]: `handle()` method doesn't really use the content of the RTU, just the related trip_updates.
So the contributor doesn't affect those tests.

Comment (mbounabi) [reply tests/integration/handle_test.py]: > BTW, the signature of handle does look a bit redundant (RTU + TU...)
Is it supposed to be changed in the refacto? If not, worth added to the ticket

Comment (pbougue) [reply tests/integration/handle_test.py]: Added to the last refactor ticket : https://jira.kisio.org/browse/NAVP-1353

**Pull request merged**